pytest.ini

```diff
@@ -26,4 +26,5 @@ env =
     D:PROVIDER_PRIVATE_KEY=0xfd5c1ccea015b6d663618850824154a3b3fb2882c46cefb05b9a93fea8c3d215
     D:ADDRESS_FILE=~/.ocean/ocean-contracts/artifacts/address.json
     D:OCEAN_NETWORK_URL=http://127.0.0.1:8545
+    D:IPFS_GATEWAY=https://api.web3.storage/upload
```

Comment:
this shouldn't be set for all tests, please set it with monkeypatch inside the needed tests.